dashboard/db/migrate/20170406215340_create_pd_payment_terms.rb

```diff
@@ -0,0 +1,14 @@
+class CreatePdPaymentTerms < ActiveRecord::Migration[5.0]
+  def change
+    create_table :pd_payment_terms do |t|
+      t.references :regional_partner, foreign_key: true
+      t.date :start_date
+      t.date :end_date
+      t.string :course
+      t.string :subject
+      t.text :properties
```

Comment:
There's no default that makes sense for any of these. Properties should be non-Null, the rest can be null. I'll make the change.